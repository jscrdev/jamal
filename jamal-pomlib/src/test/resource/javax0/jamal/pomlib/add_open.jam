{@import res:javax0/jamal/pomlib/pom.jim}{@comment

This file tests the opens and exports macros

}{opens :module:prefix:to}
{opens :module:prefix}
{exports :module:prefix:to}
{exports :module:prefix}
{#define MODULE_PREFIX=module-prefix.}
{#define PACKAGE_PREFIX=package-prefix.}
{opens :module:prefix:to}
{opens :module:prefix}
{exports :module:prefix:to}
{exports :module:prefix}