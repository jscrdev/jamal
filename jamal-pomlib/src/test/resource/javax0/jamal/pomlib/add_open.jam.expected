--add-opens module/prefix=to
--add-opens module/prefix=ALL-UNNAMED
--add-exports module/prefix=to
--add-exports module/prefix=ALL-UNNAMED


--add-opens module-prefix.module/package-prefix.prefix=to
--add-opens module-prefix.module/package-prefix.prefix=ALL-UNNAMED
--add-exports module-prefix.module/package-prefix.prefix=to
--add-exports module-prefix.module/package-prefix.prefix=ALL-UNNAMED